h2o-automl/src/main/java/ai/h2o/automl/AutoMLBuildSpec.java

```diff
@@ -79,6 +79,7 @@ public AutoMLBuildControl() {
     public String fold_column;
     public String weights_column;
     public String[] ignored_columns;
+    public boolean keep_cv_preds_models;
```

Comment:
I think we should just call this parameter `keep_cross_validation_predictions` since that's what we call it elsewhere.  This is good for consistency and then we can re-use the same docs page: http://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/keep_cross_validation_predictions.html